Cython/Utility/ModuleSetupCode.c

```diff
@@ -190,11 +194,6 @@
   #define CYTHON_USE_FREELISTS 0
 
 #elif defined(CYTHON_LIMITED_API)
```

Comment:
We might also want to switch this to `defined(Py_LIMITED_API)` by now, shouldn't we?